backend/FwLite/MiniLcm/Validators/MiniLcmValidators.cs

```diff
@@ -4,12 +4,85 @@
 
 namespace MiniLcm.Validators;
 
-public record MiniLcmValidators(IValidator<ComplexFormType> ComplexFormTypeValidator)
+public record MiniLcmValidators(
+    IValidator<ComplexFormType> ComplexFormTypeValidator,
+    IValidator<Entry> EntryValidator,
+    IValidator<Sense> SenseValidator,
+    IValidator<ExampleSentence> ExampleSentenceValidator,
+    IValidator<WritingSystem> WritingSystemValidator,
+    IValidator<PartOfSpeech> PartOfSpeechValidator,
+    IValidator<SemanticDomain> SemanticDomainValidator)
 {
-    public async Task ValidateAndThrow(ComplexFormType value)
+    public async Task ValidateAndThrowAsync(ComplexFormType value)
     {
         await ComplexFormTypeValidator.ValidateAndThrowAsync(value);
     }
+
+    public async Task ValidateAndThrowAsync(Entry value)
+    {
+        await EntryValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(Sense value)
+    {
+        await SenseValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(ExampleSentence value)
+    {
+        await ExampleSentenceValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(WritingSystem value)
+    {
+        await WritingSystemValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(PartOfSpeech value)
+    {
+        await PartOfSpeechValidator.ValidateAndThrowAsync(value);
+    }
+
+    public async Task ValidateAndThrowAsync(SemanticDomain value)
+    {
+        await SemanticDomainValidator.ValidateAndThrowAsync(value);
+    }
+
+    public void ValidateAndThrow(ComplexFormType value)
```

Comment:
I'd like to not expose these if we can avoid it, when using the non async version you can't run any async validation rules. We don't have any now but I don't want to block us from using them in the future.

Reply:
Removed the sync versions as they were no longer called once I fixed the sync blocking warnings from https://github.com/sillsdev/languageforge-lexbox/pull/1344#pullrequestreview-2535833309